locust/argument_parser.py

```diff
@@ -357,7 +357,13 @@ def setup_parser_arguments(parser):
         metavar='LocustClass',
         help="Optionally specify which Locust classes that should be used (available Locust classes can be listed with -l or --list)",
     )
-
+    parser.add_argument(
+        '-B', '--web-auth',
+        type=str,
+        dest='web_auth',
+        default=None,
+        help='Set the Basic Auth for the UI (fmt: username:password)'
+    )
```

Comment:
You should pull the latest master branch and make sure that the argument is added to the Web UI argument group.